src/main/scala/com/wavesplatform/mining/Miner.scala

```diff
@@ -99,17 +100,20 @@ class Miner(
     else if (unconfirmed.isEmpty) {
       log.trace("skipping microBlock because no txs in utx pool")
       Right(None)
-    } else {
+    }
+    else {
       log.trace(s"Accumulated ${unconfirmed.size} txs for microblock")
       (for {
         fullAndMicro <- measureSuccessful(microBlockBuildTimeStats, {
-          val signedBlock = Block.buildAndSign(version = 3,
+
+          val eitherBlock = Block.buildAndSign(version = 3,
```

Comment:
for {
signedBlock <- Block.buildAndSign
microBlock <- MicroBlock.buildAndSign
} yield (signedBlock, microBlock)